sympy/functions/special/bsplines.py

```diff
@@ -138,14 +141,7 @@ def bspline_basis_set(d, knots, x):
     >>> knots = range(5)
     >>> splines = bspline_basis_set(d, knots, x)
     >>> splines
-    [Piecewise((x**2/2, And(x < 1, x >= 0)),
-               (-x**2 + 3*x - 3/2, And(x < 2, x >= 1)),
-               (x**2/2 - 3*x + 9/2, And(x <= 3, x >= 2)),
-               (0, True)),
-     Piecewise((x**2/2 - x + 1/2, And(x < 2, x >= 1)),
-               (-x**2 + 5*x - 11/2, And(x < 3, x >= 2)),
-               (x**2/2 - 4*x + 8, And(x <= 4, x >= 3)),
-               (0, True))]
+    [Piecewise((x**2/2, (x >= 0) & (x < 1)), (-x**2 + 3*x - 3/2, (x >= 1) & (x < 2)), (x**2/2 - 3*x + 9/2, (x >= 2) & (x <= 3)), (0, True)), Piecewise((x**2/2 - x + 1/2, (x >= 1) & (x < 2)), (-x**2 + 5*x - 11/2, (x >= 2) & (x < 3)), (x**2/2 - 4*x + 8, (x >= 3) & (x <= 4)), (0, True))]
```

Comment:
maybe we can keep the line feeds, lines in the documentation examples should be shorter than 80 characters, otherwise the HTML documentation on the website will exceed the page border.